src/main/java/lotto/domain/RanksCount.java

```diff
@@ -0,0 +1,58 @@
+package lotto.domain;
+
+import lotto.enums.WinningRank;
+
+import java.util.*;
+
+public class RanksCount {
+    private static final int INIT = 0;
+    private static final int MATCHED_COUNT_MIN = 3;
+    private static final int PLUS = 1;
+    private final TreeMap<WinningRank, Integer> ranksCount;
+
+    public RanksCount() {
+        this.ranksCount = new TreeMap<>(Comparator.comparingInt(WinningRank::matchedCount));
+        init();
```

Comment:
생성자에서 가장 중요한 일은 필드에 값을 "할당"하는 일인데요, 그렇다면 "할당" 이라는 행위가 드러나도록 하는게 좋다고 생각합니다.
```suggestion
        ranksCount = initialValues();
```

Reply:
지금까지는 생성자에서 `할당`만하면 된다는 생각을 가지고 있었는데
예시처럼 확실히 `할당`이라는 행위를 나타내도록 하는 것이 어떤 부분에 할당되는지도 한 눈에 알 수 있어서 더 좋은 것 같습니다!
말씀하신 부분 반영하여 수정하였습니다!